### PR TITLE
Quick prediction

### PR DESCRIPTION
**Pull request by levkk**
1. Quick prediction module in Model page.
2. `pgml.model_predict(model_id BIGINT, features DOUBLE PRECISION[])` for predicting against a specific model for debugging.

https://user-images.githubusercontent.com/9561483/168343255-2313d545-fb15-44e8-bc6f-65573f574df0.mp4

### CODE REVIEW

File: pgml-dashboard/app/views/models.py [hunk 1]
```diff
@@ -2,11 +2,17 @@
 from collections import namedtuple
 import json
 import logging
+import pickle
```
Comment (montanalow) [inline pgml-dashboard/app/views/models.py:5]: is this actually used?

**Review by montanalow — APPROVED** [review]

**Comment by montanalow** [comment]
This reminds me, we need to add Shap to the backlog! https://github.com/slundberg/shap

File: pgml-dashboard/app/views/models.py (hunk 1 above)
Comment (levkk) [reply pgml-dashboard/app/views/models.py:5]: No, good call

Comment (levkk) [inline pgml-dashboard/app/views/models.py:5]:
```suggestion
```

File: pgml-dashboard/app/views/models.py [hunk 2]
```diff
@@ -2,11 +2,16 @@
 from collections import namedtuple
 import json
 import logging
+import io
```
Comment (levkk) [inline pgml-dashboard/app/views/models.py:5]:
```suggestion
```

**Pull request merged**